plugins/alignments/src/LinearPileupDisplay/model.ts

```diff
@@ -404,6 +405,72 @@ const stateModelFactory = (configSchema: LinearPileupDisplayConfigModel) =>
             modificationTagMap: JSON.parse(JSON.stringify(modificationTagMap)),
             showSoftClip: self.showSoftClipping,
             config: self.rendererConfig,
+            async onFeatureClick(_: unknown, featureId: string | undefined) {
+              const session = getSession(self)
```

Comment:
the onFeatureClick is now significantly more advanced here for Pileup display specifically to do async feature details